### DISCUSSION TITLE
[advanced-typeshed] Confusing hint

### DISCUSSION THREAD

**Discussion by F-park** [laike9m/Python-Type-Challenges#86] [Ideas]
https://github.com/laike9m/Python-Type-Challenges/blob/c6ca3a1be63ee4f72ff3e84cc906dcf082247de8/challenges/advanced-typeshed/solution.py#L6-L7

`Readme.md` of typeshed cannot get any hint, but I cannot find any docs about `_typeshed`, maybe should hint with `pyi`.

`https://github.com/python/typeshed` ->
`https://github.com/python/typeshed/blob/main/stdlib/_typeshed/__init__.pyi`

**Comment by laike9m** [comment]
Makes sense to me. Feel free to create a PR and I'll approve
